src/activities/CreateAreaMeasurement3D.ts

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import type {
+    IActivityHandler,
+    IActivityContext,
+} from "@geocortex/workflow/runtime/IActivityHandler";
+import { MapProvider } from "@geocortex/workflow/runtime/activities/arcgis/MapProvider";
+import { activate } from "@geocortex/workflow/runtime/Hooks";
+import AreaMeasurement3D from "@arcgis/core/widgets/AreaMeasurement3D";
+import SceneView from "@arcgis/core/views/SceneView";
+import { removeFunction, areaMeasurementUnits, AreaMeasurement3DResult } from "../types";
+
+/** An interface that defines the inputs of the activity. */
+interface CreateAreaMeasurement3DInputs {
+    /**
+     * @description Unit system (imperial or metric) or specific unit used for area values.
+     */
+    areaUnit?: areaMeasurementUnits | string;
+}
+
+interface CreateAreaMeasurement3DOutputs {
+
+    measurement: AreaMeasurement3DResult | undefined;
+    /**
+     * @description Function that removes the measurement from the map.
+     */
+    remove: removeFunction | undefined;
+
+}
+
+/**
+ * @category ArcGIS Maps SDK for JavaScript
+ * @clientOnly
+ * @defaultName measure
+ * @description Measure a distance or area on a 3D map.
```

Comment:
Should be "area" only.

Reply:
The two distance activities have the same issue too

Reply:
done